packages/loader/container-loader/src/containerStorageAdapter.ts

```diff
@@ -58,7 +59,9 @@ export class ContainerStorageAdapter implements IDocumentStorageService, IDispos
 			this.logger,
 		));
 
-		if (this.captureProtocolSummary !== undefined) {
+		this.options.summarizeProtocolTree =
+			this.options.summarizeProtocolTree ?? service.policies?.summarizeProtocolTree;
+		if (this.options.summarizeProtocolTree === true) {
```

Comment:
Do we need to have this.options.summarizeProtocolTree at all? I.e. it feels like local variable (or maybe not even having local variable) is sufficient here  (I assume no other code touches this.options.summarizeProtocolTree, but I have not checked that)